### PR TITLE
Docs: Update the doc on execution devices property and enable_startup_fallback property

### PR DESCRIPTION
**Pull request by WeldonWangwang**
### Details:
 - *ov::intel_auto::enable_startup_fallback in 92996*
 - *ov::execution_devices in 92659*

### Tickets:
 - *99209*

### CODE REVIEW

**Review by wangleis — APPROVED** [review]

**Comment by wangleis** [comment]
Need another PR for 22.3 branch

**Comment by peterchen-intel** [comment]
Waiting for the review and merge of https://github.com/openvinotoolkit/openvino/pull/14503

**Review by wangleis — APPROVED** [review]

File: docs/OV_Runtime_UG/auto_device_selection.md [hunk 1]
```diff
@@ -76,41 +76,55 @@ Following the OpenVINO™ naming convention, the Automatic Device Selection mode
...
-| |                              | |       `ov::hint::Priority::HIGH`                                   |
-| |                              | |       `ov::hint::Priority::MEDIUM`                                 |
-| |                              | |       `ov::hint::Priority::LOW`                                    |
-| |                              | |                                                                    |
-| |                              | | Indicates the priority for a model.                                |
-| |                              | | IMPORTANT: This property is not fully supported yet.               |
-+--------------------------------+----------------------------------------------------------------------+
++---------------------------------------------+----------------------------------------------------------------------+
+| | Property                                  | | Values and Description                                             |
++======================================v======+======================================================================+
```
Comment (xu-yuan1) [inline docs/OV_Runtime_UG/auto_device_selection.md:81]:
```suggestion
+=============================================+======================================================================+
```

File: docs/OV_Runtime_UG/auto_device_selection.md [hunk 2]
```diff
@@ -76,41 +76,55 @@ Following the OpenVINO™ naming convention, the Automatic Device Selection mode
...
+| |                                           | | Examples of returning results could be `(CPU)`(`(CPU)` is a        |
+| |                                           | | temporary device, indicating that CPU is used for acceleration at  |
+| |                                           | | the model compilation stage), `CPU`, `GPU`, `CPU GPU`, `GPU.0`,    |
+| |                                           | | etc.                                                               |
++---------------------------------------------+----------------------------------------------------------------------+
+| | `ov::intel_auto::enable_startup_fallback` | |  **Values**:                                                       |
+| |                                           | |       `true`                                                       |
+| |                                           | |       `false`                                                      |
+| |                                           | |                                                                    |
+| |                                           | | Enable/disable CPU as acceleration (or helper device) at the       |
```
Comment (xu-yuan1) [inline docs/OV_Runtime_UG/auto_device_selection.md:125]:
```suggestion
| |                                           | | Enables/disables CPU as acceleration (or the helper device) in the       |
```

File: docs/OV_Runtime_UG/auto_device_selection.md [hunk 3]
```diff
@@ -76,41 +76,55 @@ Following the OpenVINO™ naming convention, the Automatic Device Selection mode
...
+| |                                           | | beginning. The default value is `true`.                            |
```
Comment (xu-yuan1) [inline docs/OV_Runtime_UG/auto_device_selection.md:126]:
```suggestion
| |                                           | | beginning. The default value is `true`, indicating that CPU is used as acceleration by default.                            |
```

**Review by xu-yuan1 — APPROVED** [review]

**Comment by xu-yuan1** [comment]
With this new attribute `ov::intel_auto::enable_startup_fallback`, is the statement in Line 47 still holding true?

> What is important, **AUTO always starts inference with the CPU of the system**,

If not, we can revise it to "AUTO starts inference with the CPU of the system **by default**...", and add this information in the note in Line 52 as "Note that if you choose to exclude CPU from the priority list **or disable the initial CPU acceleration feature via `ov::intel_auto::enable_startup_fallback`, it will be unable to support the initial model compilation stage."

Besides, we also need to add information about two questions:

- Once you set it to false, will it take effect from the next inference?
- If it's set to false, how will AUTO decide to use what device?

**Comment by WeldonWangwang** [comment]
> 

Yes, agree, we can modify the description in Line 47 and will add some notes.
CPU acceleration only works in the load model stage and has no effect on next inference.
The logic of the device used in AUTO selection reasoning has not changed, which is independent of `ov::intel_auto::enable_startup_fallback`

**Review by kblaszczak-intel — APPROVED** [review]

**Pull request merged**